### PR TITLE
SBERDOMA-30 design feedback fices

### PR DESCRIPTION
**Pull request by dkoviazin**
1. Fix chrome autofill in the registration form
2. Fix antd inputs css in formItems only
3. Remove red starts from required fields in auth forms
S1
N2

### CODE REVIEW

**Review by pahaz — APPROVED** [review]

File: apps/condo/pages/auth/change-password.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const ChangePasswordPage: AuthPage = () => {
                 initialValues={initialValues}
                 colon={false}
                 style={{ marginTop: '40px' }}
+                requiredMark={false}
```
Comment (pahaz) [inline apps/condo/pages/auth/change-password.tsx:65]: is it important?

Comment (dkoviazin) [reply apps/condo/pages/auth/change-password.tsx:65]: This flag only removes a red star from labels in required fields, the field still remains required. There are 3 options and they are strange:
1. By default = red star
2. false = hidden
3. optional = add word "optional" to all not required fields labels

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 2]
```diff
@@ -5,6 +5,39 @@ import { Menu } from 'antd'
...
+    -webkit-text-fill-color: #000;
+}
+.ant-form-item-has-error .ant-input-affix-wrapper > input.ant-input:focus,
+.ant-input-affix-wrapper > input.ant-input:focus {
+    -webkit-box-shadow: inset 0 0 0 50px #fff;
+    -webkit-text-fill-color: #000;
+    background-color: white;
+}
+.ant-form-item-has-error .ant-input-affix-wrapper:not(.ant-input-affix-wrapper-disabled):hover {
+    border-color: #ff4d4f;
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:29]: use colors from constants

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 3]
```diff
@@ -5,6 +5,39 @@ import { Menu } from 'antd'
...
+}
+.ant-form-item-has-error .ant-input-affix-wrapper-focused:hover,
+.ant-form-item-has-error .ant-input-affix-wrapper:focus:hover {
+    border-color: #ff4d4f;
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:33]: use colors from constants

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts (hunk 3 above)
Comment (dkoviazin) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts]: done

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts]: done

**Pull request merged**